studyzen/FlashCard/FlashCardController.cs

```diff
@@ -0,0 +1,73 @@
+using Microsoft.AspNetCore.Mvc;
+using StudyZen.Common;
+using StudyZen.FlashCards.Requests;
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+
+
+namespace StudyZen.FlashCards;
+
+[ApiController]
+[Route("[controller]")]
+public sealed class FlashCardsController : ControllerBase
+{
+    private readonly IFlashCardService _flashCardService;
+
+    public FlashCardsController(IFlashCardService flashCardService)
+    {
+        _flashCardService = flashCardService;
+    }
+
+    [HttpPost]
+    public IActionResult CreateFlashcard([FromBody] CreateFlashCardRequest? request)
+    {
+        request = request.ThrowIfRequestArgumentNull(nameof(request));
+
+        var flashCard = _flashCardService.AddFlashCard(request);
+   
+        return CreatedAtAction(nameof(GetFlashcard), new { flashCardId = flashCard.Id }, flashCard);
+    }
+
+    [HttpGet]
+    [Route("{flashCardId}")]
+    public IActionResult GetFlashcard(int flashCardId)
+    {
+        var flashCard = _flashCardService.GetFlashCard(flashCardId);
+        
+        return flashCard == null ? NotFound() : Ok(flashCard);
+    }
+    
+
+    [HttpGet]
+    public async Task<IActionResult> GetFlashCardsBySetId(int? flashCardSetId)
+    {
+        return Ok(_flashCardService.GetFlashCardsBySetId(flashCardSetId));
+    }
+    
+
+    [HttpDelete("{flashCardId}")]
+    public IActionResult DeleteFlashcard(int flashCardId)
+    {
+        var deleted = _flashCardService.DeleteFlashCard(flashCardId);
+        
```

Comment:
just NoContent()

Reply:
somehow I believe that you reviewed not the latest version at that time because these fixes were already implemented on Friday